packages/dds/sequence/src/sequence.ts

```diff
@@ -711,9 +711,79 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 
 	/**
 	 * {@inheritDoc @fluidframework/shared-object-base#SharedObjectCore.applyStashedOp}
+	 * @internal
 	 */
-	protected applyStashedOp(content: any): unknown {
-		return this.client.applyStashedOp(parseHandles(content, this.serializer));
+	protected applyStashedOp(
+		content: any,
+	): IMapMessageLocalMetadata | SegmentGroup | SegmentGroup[] {
+		const intervalContent: IIntervalDeltaOp =
+			this.intervalCollections.tryGetStashedOpLocalMetadata(content) as IIntervalDeltaOp;
+		const collection = this.getIntervalCollection(content.key);
+		let metadata: IMapMessageLocalMetadata | SegmentGroup | SegmentGroup[];
+		let interval: SequenceInterval | undefined;
+		switch (intervalContent.opName) {
```

Comment:
please put the interval handling somewhere, maybe function? or like how we do it for resubmit?